### ISSUE TITLE
ArmV7 - Performance Issue

### ISSUE THREAD

**Issue by Cicorione** [Drive4ik/simple-tab-groups#1010]
Dear Drive4ik,

On Armbian and  Odroid-XU4 (armv7/armhf) STG consumes in idle 30% of the CPU (through Firefox). Unfortunately I had to disable it since the temperature was too way high.

I  understand the CPU target may not be in your priority list but it would be cool if can improve the performance on ArmV7.

```
Distributor ID:	Debian (Armbian)
Description:	Debian GNU/Linux 11 (bullseye)
Release:	11
Codename:	bullseye
Linux odroidxu4 5.4.225-odroidxu4 #22.11.1 SMP PREEMPT Wed Nov 30 10:55:16 UTC 2022 armv7l GNU/Linux
Mozilla Firefox 102.7.0esr
Simple-tab-groups: 4.7.2.1
```

Thanks,
C.

**Comment by nekohayo** [comment]
It shouldn't, normally? At least on my x86_64 computers, it uses 0% CPU when I'm not actively doing something with it. Many websites use the CPU all the time (such as YouTube and Reddit) but STG doesn't do that on my end. Is ARMv7 _that weak_ in terms of performance?

I presume you have checked `about:performance`, and used the Firefox profiler tool? What tells you that STG is the culprit? Please share the data or profiling output you have.

**Comment by Cicorione** [comment]
I investigated further this issue, and I noticed it was caused by the interaction between Firefox "Facebook Container" (FC) extensions and STG, but while the former from the performance tab didn't seem consume CPU the latter was constantly at 30% of CPU usage.

Disabling FC fix resolved the CPU consuming, and now it is at 0%, I uninstalled the FC extensions since I don't even have Feisbook account, and to avoid furher interaction with other extension I disable Firefox Container.

So far I haven't noticed other unexpected behavior from this great extension!

Thanks,

F.
